docker/docker_image_src.go

```diff
@@ -83,7 +83,14 @@ func (s *dockerImageSource) fetchManifest(tagOrDigest string) ([]byte, string, e
 	url := fmt.Sprintf(manifestURL, s.ref.ref.RemoteName(), tagOrDigest)
 	headers := make(map[string][]string)
 	headers["Accept"] = s.requestedManifestMIMETypes
-	res, err := s.c.makeRequest("GET", url, headers, nil)
+	opts := requestOptions{
+		scope: authScope{
+			remoteName: s.ref.ref.RemoteName(),
+			actions:    "pull",
+		},
+		headers: headers,
+	}
+	res, err := s.c.makeRequest("GET", url, opts)
```

Comment:
I wouldn’t be opposed to (but won’t insist on) inlining this:
```go
res, err := s.c.makeRequeset("GET", url, requestOptions{
       …
   })
```
(and the same throughout, if `scope` is left anywhere.)